packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx

```diff
@@ -224,18 +229,24 @@ const GraphicalRecording: React.FC = () => {
               options={[
                 {
                   text: fullName(graphicalRecordingMode.pairing.players[0]),
-                  onPress: handleSelectWinner,
+                  onPress: () =>
+                    handleSelectWinner(
+                      graphicalRecordingMode.pairing.players[0],
+                    ),
                 },
                 {
                   text: fullName(graphicalRecordingMode.pairing.players[1]),
-                  onPress: handleSelectWinner,
+                  onPress: () =>
+                    handleSelectWinner(
+                      graphicalRecordingMode.pairing.players[1],
+                    ),
                 },
               ]}
               visible={showEndGame}
               onCancel={cancelSelection}
```

Comment:
Would be nice to put these options at the top as a variable called winnerSelectionOptions to be consistent (the action buttons and promotion buttons are both under the buttom parameters comment on line 85), otherwise our view gets super messy with all these sheet options!

Reply:
How badly do you want these not to be here because I can't just have it as a variable (it complains that graphicalRecordingMode could be undefined) and from some investigation, I think it would be a pain to implement it elsewhere because of the way option sheet is set up

Reply:
hmmm i guess it doesnt matter that much its just to keep it clean, you can do something like this though:
![image](https://user-images.githubusercontent.com/26475724/178134761-b9abccd1-4d6d-482f-9e27-b68eeb3ba9fb.png)

Reply:
oh cool I didn't know I could do that (my lack of react experience is showing), I fixed everything else so I'll update the pr. (Also stop doing code on your birthday, do and do something fun :) )